fastlane/Fastfile

```diff
@@ -57,7 +57,7 @@ platform :ios do
       ],
       export_options: {
         provisioningProfiles: {
-          "edu.stanford.spezi.templateapplication" => "Spezi Template Application"
+          "edu.stanford.BDHG.MyHeartCounts" => "My Heart Counts"
```

Comment:
I will update this in the future once the CI is working and everything is passing there so I can run an test the TestFlight deployment 👍